core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -147,12 +147,13 @@
           </a>
         </li>
         <li class="nav-item" ng-if="$ctrl.isSuperAdmin">
-          <a class="dropdown-item nav-link protractor-test-admin-link" ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/admin" \
-             target="_blank" rel="noopener">
+          <a class="dropdown-item nav-link protractor-test-admin-link" ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/admin" target="_blank" rel="noopener">
```

Comment:
You can still break the line at "target" to keep it within 80 chars. You just don't need a backslash at the end to do that, in HTML.

When you do so, align the left character of "target" with the left character of "class" in the previous line.

Reply:
Done PTAL Thanks!